sepal_ui/mapping/aoi_control.py

```diff
@@ -0,0 +1,167 @@
+import ee
+from shapely import geometry as sg
+from ipyleaflet import WidgetControl
+
+from sepal_ui.mapping.map_btn import MapBtn
+from sepal_ui import sepalwidgets as sw
+from sepal_ui.scripts import utils as su
+
+
+class AoiControl(WidgetControl):
+    """
+    Widget control providing zoom options for the end user. The developer can add as many gemetries to the widget and the user will simply have to click on them to move to the appropriate AOI.
+
+    Args:
+        m (ipyleaflet.Map): the map on which he AoiControl is displayed to interact with itszoom and center
+    """
+
+    m = None
+    "(ipyleaflet.Map): the map on which he AoiControl is displayed to interact with itszoom and center"
+
+    aoi_list = None
+    "(list): the list of ListItem used in the menu. each one as the name provided by the user and the value of the bounds"
+
+    aoi_bounds = None
+    "(dict): the list of bounds to fit on. using the aoi names as keys"
+
+    def __init__(self, m, **kwargs):
+
+        # load the map
+        self.m = m
+
+        # init the aoi data list
+        self.aoi_bounds = {}
+
+        # set some default parameters
+        kwargs["position"] = kwargs.pop("position", "topright")
+
+        # create a hoverable btn
+        btn = MapBtn(logo="fas fa-at", v_on="menu.on")
```

Comment:
What about `search-location` or `search-plus`? `@` (as always) is not so intuitive for me.

Reply:
`search-location` makes complete sense and is aligned with what they use in the main framework